### DISCUSSION TITLE
Cypress test runner uses older Firefox version

### DISCUSSION THREAD

**Discussion by beatefalconer** [cypress-io/cypress#21502] [Questions and Help] [answered]
I have the latest Firefox version installed (currently 100). Cypress complains that I have version 52 installed. Does anyone know how to fix this?

![image](https://user-images.githubusercontent.com/17054469/168402812-c24b30af-5199-4c45-9d01-176b1f676faa.png)

**Comment by gplusdotgr** [comment, marked as answer]
@beatefalconer you probably have multiple versions instaleld in your system. I see you're on Windows, are you on the Linux Subsystem VM, or native Windows? (might be there's two separate Firefoxes installed). 
There's clearly an old FF version leftover that Cypress is finding, that's for sure, so I would look in your system to see if you can locate where/what it is. (but let us know if you use the Linux Subsystem)

**Reply by beatefalconer** [reply]
Yes that was the problem, I had Firefox 100 and 52 installed. I removed 52 and Cypress automatically found 100. Thanks!

![image](https://user-images.githubusercontent.com/17054469/168907231-f38ee80e-99b3-4df3-a109-65f314801fe4.png)
